client/my-sites/plugins-wpcom/premium-plugins-panel.jsx

```diff
@@ -1,72 +1,74 @@
+/**
+ * External dependencies
+ */
 import React, { PropTypes } from 'react';
 import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import {
+	flowRight as compose,
+	identity,
+} from 'lodash';
 import classNames from 'classnames';
 
+/**
+ * Internal dependencies
+ */
 import Card from 'components/card';
 import SectionHeader from 'components/section-header';
 import PurchaseButton from './purchase-button';
 import { recordTracksEvent } from 'state/analytics/actions';
 
 import Plugin from './plugin';
 
-export const PremiumPluginsPanel = React.createClass( {
-	render() {
-		const {
-			isActive = false,
-			onClick,
-			purchaseLink,
-			plugins = []
-		} = this.props;
+export const PremiumPluginsPanel = ( {
+	isActive = false,
+	onClick,
+	purchaseLink,
+	plugins = [],
+	translate = identity,
+} ) => (
+	<div>
+		<SectionHeader label={ translate( 'Premium Plan Upgrades' ) }>
+			<PurchaseButton { ...{ isActive, href: purchaseLink } } />
+		</SectionHeader>
 
-		const cardClasses = classNames( 'wpcom-plugins__premium-panel', {
-			'is-disabled': ! isActive
-		} );
-
-		return (
-			<div>
-				<SectionHeader label={ this.translate( 'Premium Plan Upgrades' ) }>
-					<PurchaseButton { ...{ isActive, href: purchaseLink } } />
-				</SectionHeader>
-
-				<Card className={ cardClasses }>
-					<div className="wpcom-plugins__list">
-						{ plugins.map( ( { name, descriptionLink, icon, category, description } ) =>
-							<Plugin
-								onClick={ () => onClick( name ) }
-								{ ...{
-									name,
-									key: name,
-									descriptionLink,
-									icon,
-									isActive,
-									category,
-									description
-								} }
-							/>
-						) }
-					</div>
-				</Card>
+		<Card className={ classNames( 'wpcom-plugins__premium-panel', {
+			'is-disabled': ! isActive,
+		} ) }>
+			<div className="wpcom-plugins__list">
+				{ plugins.map( ( { name, descriptionLink, icon, category, description } ) =>
+					<Plugin
+						{ ...{
+							category,
+							description,
+							descriptionLink,
+							icon,
+							isActive,
+							key: name,
+							name,
+							onClick,
+						} }
```

Comment:
@Copons you'll notice that I simply moved the closure from here to the Plugin component